plugins/msg-box-info/pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.pentaho.di.plugins</groupId>
+    <artifactId>pdi-plugins</artifactId>
+    <version>10.1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>msg-box-info</artifactId>
+  <version>10.1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>PDI Msg Box Info Plugin</name>
+  <description>This plugin is for pentaho Data integration (ETL) a.k.a kettle</description>
+  <url>http://www.pentaho.com/</url>
+
+  <modules>
+    <module>impl</module>
+    <module>ui</module>
+    <module>assemblies</module>
+  </modules>
+
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+
+  <properties>
+    <org.eclipse.swt.version>4.6</org.eclipse.swt.version>
+    <jface.version>3.3.0-I20070606-0010</jface.version>
```

Comment:
We probably want to move this version into the kettle parent pom and remove it from this pom and the ui pom.